server/segment_rpc.go

```diff
@@ -634,6 +635,15 @@ func genPayment(sess *BroadcastSession, numTickets int) (string, error) {
 		}
 
 		protoPayment.TicketSenderParams = senderParams
+
+		ratPrice, _ := ratPriceInfo(protoPayment.ExpectedPrice)
+		glog.V(common.VERBOSE).Infof("Created new payment - recipient=%v  faceValue=%v  winProb=%v  price=%v  numTickets=%v",
+			ethcommon.BytesToAddress(sess.OrchestratorInfo.GetTicketParams().GetRecipient()),
+			eth.FormatUnits(batch.FaceValue, "ETH"),
+			new(big.Rat).Mul(batch.Tickets()[0].WinProbRat(), big.NewRat(100, 1)).FloatString(10)+" %",
```

Comment:
Since we're logging the win prob as a value 0.0-1.0 on the orchestrator I think we could just stick with using a value 0.0-1.0 here.

Reply:
fixed in 1a02391